tmdb-api/src/commonMain/kotlin/app/moviebase/tmdb/model/TmdbPeopleModel.kt

```diff
@@ -112,13 +112,13 @@ data class TmdbPersonDetail(
 
 @Serializable
 data class TmdbCrew(
-    @SerialName("adult") val adult: Boolean,
-    @SerialName("gender") val gender: TmdbGender,
-    @SerialName("id") override val id: Int,
+    @SerialName("adult") val adult: Boolean = false,
+    @SerialName("gender") val gender: TmdbGender = TmdbGender.UNKNOWN,
+    @SerialName("id") override val id: Int = 0,
```

Comment:
It would be better to use `null` values instead of empty data, otherwise, apps need to check this specific value.